accord-maelstrom/src/main/java/accord/maelstrom/Cluster.java

```diff
@@ -163,15 +163,17 @@ public boolean processPending()
                     }
                     err.println(clock++ + " RECV " + deliver);
                     err.flush();
-                    if (deliver.body.in_reply_to > Body.SENTINEL_MSG_ID)
+                    Object body = ((Wrapper)deliver.body).body;
+                    //TODO : REVIEW : Don't know the point of deliver.body.in_reply_to > Body.SENTINEL_MSG_ID, but without it InformOfTxnReply fails
+                    if (deliver.body.in_reply_to > Body.SENTINEL_MSG_ID || body instanceof Reply)
```

Comment:
Don't know the point of deliver.body.in_reply_to > Body.SENTINEL_MSG_ID, but without it InformOfTxnReply fails

Reply:
leaving as is, moved TODO to a comment documenting the RHS of the else